lib/deep-linking/actions.js

```diff
@@ -36,6 +45,23 @@ function openHashedForm(store, urlProps) {
   }
 
   if (find(`[${refAttr}="${uri}"]`) && el) {
+    // el is in the body, not the head. check to see if we have permission to edit it
+    const isPageComponent = isComponentInPage(uri);
+
+    if (!isPageComponent && !isAdmin) {
+      // layout component that cannot be edited! display a warning to the user
+      const name = label(getComponentName(uri));
+
+      store.dispatch('addAlert', { type: 'error', text: `You do not have permission to edit this "${name}". Please contact an admin for assistance.` });
```

Comment:
Is `admin` a concept a writer/user would recognize? Would they know this means an editor? Or would they just go into #help

Reply:
Not entirely sure. I think some of them (copyeditors) are familiar, but I'm fine if they go into #help. Also the wording will definitely be tweaked as we get user feedback.